trader/trader.go

```diff
@@ -123,12 +123,12 @@ func (t *Trader) Start() {
 	for {
 		currentUpdateTime := time.Now()
 		if lastUpdateTime.IsZero() || t.timeController.ShouldUpdate(lastUpdateTime, currentUpdateTime) {
-			success := t.update()
+			updateResult := t.update()
```

Comment:
since the updateResult is returned by this update() function, the struct should be specified in this file trader.go (repeating this comment from above to as a point of reference for usage of this struct here)

Reply:
This will create an import cycle, since `trader.go` already imports `plugins`, and we pass a `UpdateLoopResult` to `SendUpdateEvent`.

Reply:
ok good point. Then let's leave as-is. Can you add a comment on the declaration of UpdateLoopResult in plugins/metricsTracker pointing this out as the reason why we define it there as opposed to in trader.go?